test/integration/analytical_policy.hpp

```diff
@@ -704,32 +701,32 @@ void test_analytical_ternary_chemical_activation(
   }
 
   std::vector<std::string> header = { "time", "A", "B", "C" };
-  writeCSV("analytical_concentrations.csv", header, analytical_concentrations, times);
-  writeCSV("model_concentrations.csv", header, model_concentrations, times);
+  writeCSV("tca_analytical_concentrations.csv", header, analytical_concentrations, times);
+  writeCSV("tca_model_concentrations.csv", header, model_concentrations, times);
```

Comment:
is writing the results to csv files needed for the test? if so, does this prevent us from running tests in parallel?

Reply:
it's not needed. We can run them in parallel and they'll overwrite each others results but these are only used when looking at the output. I changed the name so that other tests don't overwrite these results